mitodl/mitxpro
# Updated login/registration styling

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #263 

#### What's this PR do?
- Updates the login and signup UI/styling
- Adds a notification when a user enters their email to sign up and submits

#### How should this be manually tested?
- Go through the login and signup flows, make sure the UI looks good and matches the designs
- Make sure the "email sent" notification in the signup flow behaves as expected 

#### Screenshots (if appropriate)
<img width="955" alt="ss 2019-05-21 at 15 17 52 " src="https://user-images.githubusercontent.com/14932219/58124229-bd919600-7bdb-11e9-8a2f-8785350f35ed.png">
<img width="396" alt="ss 2019-05-21 at 15 18 10 " src="https://user-images.githubusercontent.com/14932219/58124230-bd919600-7bdb-11e9-9da0-7ed8c2101251.png">
<img width="951" alt="ss 2019-05-21 at 12 32 43 " src="https://user-images.githubusercontent.com/14932219/58124258-d39f5680-7bdb-11e9-9308-1bd5f29a8d4e.png">
<img width="947" alt="ss 2019-05-21 at 12 32 59 " src="https://user-images.githubusercontent.com/14932219/58124259-d39f5680-7bdb-11e9-90f3-a370b8d14a2e.png">
<img width="369" alt="ss 2019-05-21 at 13 09 20 " src="https://user-images.githubusercontent.com/14932219/58124300-eb76da80-7bdb-11e9-8f88-03969c70a29a.png">
<img width="1134" alt="ss 2019-05-21 at 12 53 23 " src="https://user-images.githubusercontent.com/14932219/58124310-efa2f800-7bdb-11e9-9c51-008cf0643129.png">
<img width="358" alt="ss 2019-05-21 at 12 53 34 " src="https://user-images.githubusercontent.com/14932219/58124326-f29de880-7bdb-11e9-9fe9-357bddd9302a.png">
